### ISSUE TITLE
Unable to refresh resource bigQuery

### ISSUE THREAD

**Issue by imkhan1104** [GoogleCloudPlatform/terraformer#539]
After running the import command for resources=bigQuery i received errors for all my tables. i have permissions to view and edit the tables. 
```
2020/07/02 11:10:35 ERROR: Unable to refresh resource tfer--tb_1
2020/07/02 11:10:35 ERROR: Unable to refresh resource tfer--tb_2
2020/07/02 11:10:35 ERROR: Unable to refresh resource tfer--tb_3
```

**Comment by linuxbandit** [comment]
This just happened for me as well, the resource is GCS. 

I had already successfully imported infrastructure (GCS), I am only re-importing because I changed provider version. Could it be some outage on GCP side?

**Comment by sergeylanzman** [comment]
@imkhan1104 Try use last version of GCP provider, 
What version(terraformer + gcp provider) you use now ?

**Comment by imkhan1104** [comment]
hi @sergeylanzman i am using terraformer version 0.8.6, gcp provider version 3.28 and runnning on windows. Thanks for responding!

**Comment by sergeylanzman** [comment]
use last version of terraformer(0.8.8), it's fixed in https://github.com/GoogleCloudPlatform/terraformer/commit/ca695b3778e8845be1ab850e1518fc954e602ed4

**Comment by imkhan1104** [comment]
> use last version of terraformer(0.8.8), it's fixed in [ca695b3](https://github.com/GoogleCloudPlatform/terraformer/commit/ca695b3778e8845be1ab850e1518fc954e602ed4)

Hi can you help to release a chocolatey package update as well? thank you in advance
